extension/api.js

```diff
@@ -36,3 +36,20 @@ HideFilesOnGitHub.storage = {
 		chrome.storage.sync.set(object);
 	}
 };
+
+// Inlined partial `escape-goat` package
+window.escape = input => input
```

Comment:
You should probably not override the `escape` global.
```suggestion
window.escapeHtml = input => input
```

Reply:
Doesn't really need to be on `window` at all as it's only used in this file.